prime-router/src/main/kotlin/SettingsProvider.kt

```diff
@@ -48,6 +48,7 @@ enum class CustomerStatus {
 enum class Topic(@JsonValue val jsonVal: String, val isUniversalPipeline: Boolean = true) {
     FULL_ELR("full-elr", true),
     ETOR_TI("etor-ti", true),
+    ELR_ELIMS("elr-elims", true),
```

Comment:
I think the idea here is slightly off. The topic should be "CDC-ELIMS" because this topic is to be specific for the CDC ELIMS use case/work flow. This use case, at this time, has the following special requirements that a topic allows us to meet easier:
1. Observability: We need to write a query to answer this question: For a particular STLT (RS receiver), is there a connection set up for that STLT to receive both test messages and production messages from CDC ELIMS? This is much more doable with a topic and something like a "elr-elims" topic that is used by other senders would not allow us to do
2. CDC ELIMS as system of record: For the pilot effort, the CDC ELIMS team wants their system to be the "system of record", meaning all messages that we deliver to a STLT should be sent to the configured CDC receiver so the CDC ELIMS team can run automatic comparisons on the data they sent us vs the data we sent to the STLT. This is probably doable without a topic (we can filter on the system name in the MSH) 
3. Partitioning data streams: For this pilot, we want to ensure that existing receivers DO NOT receive this data since we will have to work with each STLT that is to be onboarded to verify they accept the messages CDC ELIMS is sending (although the pilot is currently set to be staging only, we want to have this guard rail in place in case CLIA compliance clears and we can go to production).

I'm hoping we can find a better solution to these issues during the pilot (like ReportStream being the system of record and not having to loopback messages to a CDC ELIMS receiver), but these are the needs at the moment. Indeed, only item one above really actually depends on the topic concept and we should look to figure out a better long term solution.

I understand wanting to create a generic topic like "ELR_ELIMS" that can be used more broadly, but I'm not sure I see the value of such a topic. "ELR_ELIMS" is just "FULL_ELR" and I think we can just use the FULL_ELR topic for all those cases. If we find that we can't, then perhaps we can consider a "ELR_ELIMS" topic.

Reply:
Just remembered this conversation, @brick-green was this resolved offline, or does this need to be talked through this again?

Reply:
We're good here @jimmyfagan. Appreciate the follow up!!